be/app/Http/Controllers/LessonController.php

```diff
@@ -51,7 +51,11 @@ public function store(StoreLessonRequest $request)
      */
     public function show($id)
     {
-        //
+        $lesson = Lesson::find($id);
+        return response([
+            'status' => 200,
```

Comment:
- no need to return `status` code. By default laravel will return `200` response code (to confirm it, test it with Postman)